examples/messenger.js

```diff
@@ -102,15 +102,24 @@ const findOrCreateSession = (fbid) => {
 
 // Our bot actions
 const actions = {
-  send({sessionId}, {text}) {
+  send({sessionId}, response) {
     // Our bot has something to say!
     // Let's retrieve the Facebook user whose session belongs to
     const recipientId = sessions[sessionId].fbid;
     if (recipientId) {
       // Yay, we found our recipient!
+      
+      if (response.quickreplies) { // Wit.ai wants us to include quickreplies, alright!
+				response.quick_replies = []; // The quick reply object from Wit.ai needs to be renamed.
```

Comment:
nit: we can rewrite this to be in a bit more of a function style, like so.

first, instead of mutating response, we can build up the data object.

```
const data = {};

if (data.text) {
   data.text = response.text;
}
```

then, for quickreplies, we can use `map`

```
if (response.quickreplies) {
  data.quick_replies = response.quickreplies.map(function (title) { 
    return {title: title, ...}
  })
```